README.md

```diff
@@ -1102,6 +1103,33 @@ frame.at_css("//a[text() = 'Log in']") # => Node
 #### select
 
 
+## Tracing
+
+You can use `tracing.start` and `tracing.stop` to create a trace file which can be opened in Chrome DevTools or [timeline viewer](https://chromedevtools.github.io/timeline-viewer/).
+
+```ruby
+browser.tracing.start(path: "trace.json")
+browser.go_to("https://www.google.com");
+browser.tracing.stop
+```
+
+#### tracing.start(trace_params:, \*\*options) : `Hash` | `Hash`
+
+* trace_params `Hash` 
```

Comment:
That's already clear that these are params for a trace, `trace_params` is redundant

Reply:
yeah, that's a good point, actually, my intention was to allow customize any racing params that are passed to `Tracing.start` command, I mean those:
![Selection_996](https://user-images.githubusercontent.com/23525618/145474785-1c4ec49b-2449-4fad-b94d-d98919b29354.jpg)

In general, I think we can keep only our own params because if someone really needs the tiny trace-params customization he is always able to implement his own tracing class for this, so let's drop `trace_params`